### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by Mrs4s** [Mrs4s/go-cqhttp#2043] [General]
## Changelog
本版本主要是修复错误.
本次更新后登录将不在依赖 `captcha.go-cqhttp.org` 域名. (大部分情况下)
无法连接该域名的情况下登录也将不会出现 `45` 错误.
建议请优先使用 `Android Phone` 和 `aPad` 协议密码登录.

## 修复
008d546 修复发送 `forward` 信息可能会出现 `forward 消息中必须包含 id` 错误的问题.
43ff36e 修复初始化设备信息时可能出现崩溃的问题.
43c6e3d 修复发送视频调用 `ffmpeg` 时可能出现 `cannot run executable found relative to current directory` 错误的问题.
749cde2 修复被拉进群时有可能出现报错的问题.

## 其他
更新默认 `Android Phone` `aPad` 协议到 `8.9.35.10440` 版本
修改自动更新逻辑为启动时检查, 手动更新需要添加 `-update-protocol` 参数启动

<hr /><em>This discussion was created from the release <a href='https://github.com/Mrs4s/go-cqhttp/releases/tag/v1.0.0'>v1.0.0</a>.</em>

**Comment by WJZ-P** [comment]
登录失败: 登录失败，建议升级最新版本后重试，或通过问题反馈与我们联系。 Code: 45

**Reply by WJZ-P** [reply]
作者你好，我已更新到1.0.0版本了，也调用了update-protocal参数，但是还是45错误代码，这是什么情况呢？

**Reply by WJZ-P** [reply]
已解决，使用手表协议，注意config里面密码必须留空，否则无法扫码哦！

**Reply by ghost** [reply]
使用手表协议无法使用http啊

**Comment by Bronya9425** [comment]
作者您好，我已更新至1.0.0，同样出现和上面的问题。
登录失败: 登录失败，建议升级最新版本后重试，或通过问题反馈与我们联系。 Code: 45

**Reply by Bronya9425** [reply]
已解决，感谢WJZ-P的提醒，更换协议已可以正常登陆，更换的协议与版本为iPad 8.9.33.614，使用密码登陆，未使用扫码

**Reply by kingerm** [reply]
大佬您好，请问如何更换协议与版本呢？

**Reply by maborosh** [reply]
> 大佬您好，请问如何更换协议与版本呢？

https://docs.go-cqhttp.org/guide/config.html#%E8%AE%BE%E5%A4%87%E4%BF%A1%E6%81%AF

**Reply by WJZ-P** [reply]
> > 大佬您好，请问如何更换协议与版本呢？
> 
> https://docs.go-cqhttp.org/guide/config.html#%E8%AE%BE%E5%A4%87%E4%BF%A1%E6%81%AF

你好，是在device里面更改protocol协议，不是在config里面，我说错了

**Reply by kingerm** [reply]
> > 大佬您好，请问如何更换协议与版本呢？
> 
> https://docs.go-cqhttp.org/guide/config.html#%E8%AE%BE%E5%A4%87%E4%BF%A1%E6%81%AF

感谢大佬不吝赐教！

**Reply by kingerm** [reply]
> > > 大佬您好，请问如何更换协议与版本呢？
> > 
> > 
> > https://docs.go-cqhttp.org/guide/config.html#%E8%AE%BE%E5%A4%87%E4%BF%A1%E6%81%AF
> 
> 你好，是在device里面更改protocol协议，不是在config里面，我说错了

感谢大佬！！！

**Comment by blzckcxt** [comment]
我有一个问题，现在调用的API好像是GPT2.0模型，如何调用GPT4的API呢
